---
title: "Lecture13markdown"
author: "Cody Ocheltree"
date: "5/16/2018"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

## Lecture 13 - Analysis of SNPs and Genotypes from online datasets

```{r}
snp_genotypes <- read.csv("~/Desktop/bggn213/Bggn213_github/lecture13/373531-SampleGenotypes-Homo_sapiens_Variation_Sample_rs8067378 (1).csv")
```

```{r}
table(snp_genotypes[,2]) / nrow(snp_genotypes) * 100
```


Sequencing in R

```{r}
library(seqinr)
library(gtools)
phred <- asc( s2c("DDDDCDEDCDDDDBBDDDCC@") ) - 33
phred
## D D D D C D E D C D D D D B B D D D C C @
## 35 35 35 35 34 35 36 35 34 35 35 35 35 33 33 35 35 35 34 34 31
prob <- 10**(-phred/10)
prob
```

RNA Sequencing Results

```{r}
seq_results <- read.table("rs8067378_ENSG00000172057.6.txt")
summary(seq_results)
```

```{r}
summary(seq_results$exp[seq_results$geno == "G/G"])
summary(seq_results$exp[seq_results$geno == "A/G"])
summary(seq_results$exp[seq_results$geno == "A/A"])
```

```{r}
GG <- seq_results$exp[seq_results$geno == "G/G"]
AG <- seq_results$exp[seq_results$geno == "A/G"]
AA <- seq_results$exp[seq_results$geno == "A/A"]

boxplot(GG, AG, AA, names = c("G/G", "A/G", "A/A"), notch = T)
```

```{r}
boxplot()
```

